src/extensions/padding-controls/index.js

```diff
@@ -63,15 +63,7 @@ const withPaddingControls = createHigherOrderComponent( ( BlockEdit ) => {
 		const { name, attributes, setAttributes, isSelected } = props;
 		const { padding, paddingCustom } = attributes;
 
-		const getMaxValue = () => {
-			const { maxValue = null } = props;
-
-			if ( ! maxValue ) {
-				return attributes.align === 'full' ? 10 : 5;
-			}
-
-			return maxValue;
-		};
+		const maxValue = attributes.align === 'full' ? 10 : 5;
```

Comment:
It looks like you're removing the possibility to pass a maxValue through props entirely. I would have instead changed it like this:
```
// From this:
const getMaxValue = () => {
	const { maxValue = null } = props;

// To this
const getMaxValue = ( maxValue = null ) => {
```

and then just pass the prop into the function where it's called:

```
max={ getMaxValue( props.maxValue ) }
```

If we don't need it that's fine, I just wanted to call it out.

Reply:
Good call. You are right that the ability to use the maxValue as a prop has been removed. I made this decision due to the use-case for this extension.

This was originally based on work done for the GutterControl component 
https://github.com/godaddy-wordpress/coblocks/blob/7d4333c2f9c1d7691cf8d9d1f8df144d019db5d8/src/components/gutter-control/gutter-control.js#L35-L43

In that instance, a getter is used to set a class property. Given the re-usability of the component, it makes more sense to allow an override to this dynamic property.

Reply:
I see now, thanks!